packages/webapp/src/pages/Connection/Create.tsx

```diff
@@ -205,14 +205,19 @@ export default function IntegrationCreate() {
                 content_url: contentUrl
             };
         }
-
-        nango[authMode === 'NONE' ? 'create' : 'auth'](target.integration_unique_key.value, target.connection_id.value, {
+        const connectionConfig = {
             user_scope: authMode === 'NONE' ? undefined : selectedScopes || [],
             params,
             authorization_params: authorizationParams || {},
             hmac: hmacDigest || '',
             credentials
-        })
+        };
+        const getConnection =
+            authMode === 'NONE'
+                ? nango.create(target.integration_unique_key.value, target.connection_id.value, connectionConfig)
+                : nango.auth(target.integration_unique_key.value, target.connection_id.value, connectionConfig);
+
+        getConnection
```

Comment:
the compiler doesn't like the dynamic index method calling (I don't know how this is called). No logic has changed here